python/tests/test_constraints.py

```diff
@@ -1139,9 +1139,9 @@ def test_constr_LBOUND(self):
 
             for t in range(self.T):
                 for bus in net.buses:
-                    self.assertEqual(u[bus.index_v_mag[t]],bus.v_max)
+                    self.assertEqual(u[bus.index_v_mag[t]],bus.v_max_reg)
                     self.assertEqual(u[bus.index_v_ang[t]],pf.BUS_INF_V_ANG)
-                    self.assertEqual(l[bus.index_v_mag[t]],bus.v_min)
+                    self.assertEqual(l[bus.index_v_mag[t]],bus.v_min_reg)
```

Comment:
I thought we agreed I would leave things as they currently were, which in my interpretation was that since only `v_min` existed which was set only from a regulated voltage I thought. 

I will update to `v_min_norm`